tests/integration/bond_test.py

```diff
@@ -369,3 +381,10 @@ def test_bond_with_empty_ipv6_static_address(eth1_up):
         assertlib.assert_state(bond_state)
 
     assertlib.assert_absent('bond99')
+
+
+def test_create_vlan_as_slave_of_bond(bond99_vlan101):
+    bond_name = BOND99
+    slaves = [bond99_vlan101[Interface.KEY][0][Interface.NAME]]
```

Comment:
`bond99_vlan101` represents two interfaces: a VLAN and its base, so how do you know what is defined at position `0`?